extensions/amp-story/1.0/amp-story.js

```diff
@@ -308,6 +308,7 @@ export class AmpStory extends AMP.BaseElement {
     this.localizationService_
         .registerLocalizedStringBundle('en-xa', enXaPseudoLocaleBundle);
 
+    this.continueAnyway = false;
```

Comment:
New instance vars should have JSDoc with a type declaration. However, I think we can avoid using an ivar here with my suggestion below.